### ISSUE TITLE
Dev: WAVE Accessibility: Account Management Pages, Missing Form Labels

### ISSUE THREAD

**Issue by rayneng** [hackforla/tdm-calculator#2573]
### Overview
We need to fix missing form labels across the Account Management pages, which are preventing assistive technologies from describing form controls to sight-impaired users.

### Action Items
- [ ] Add `aria-label` to all the forms that don't have a form label
  - [ ] Ticket is still `WIP`

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - ~[Staging slide, 2573](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3691501d13a_2_41#slide=id.g3691501d13a_2_41)~
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

#### Developer Resources

<details><summary>Testing Resources</summary>
<p>

- Dev: https://tdm-dev.azurewebsites.net/calculation/1/0
- Prod: https://tdm.ladot.lacity.org/login
- [WAVE Chrome extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

</p>
</details>

<details><summary>Definitions and Shorthand Terms</summary>
<p>

Term | Defined by WAVE | Definition
-- | -- | --
Form Label | Y | a `<label>` that is associated with a form control. In order to meet WAVE Tool criteria, the label.for  attribute must match the input.id attribute. Often referred to as a `<label>`, `<input>` pair.
Form Control | Y(?) | elements used in forms that allow users to input data, make selections, and interact with the form. In this ticket the form control is always an `<input>` element.

##### Visual Guide
`[todo - do we need one?]`

</p>
</details>

**Comment by ryanfchase** [comment]
Closing this ticket, it was created before we outlined our plans in the epic:
- https://github.com/hackforla/tdm-calculator/issues/2410
